base.yml

```diff
@@ -26,3 +26,7 @@ services:
     image: "selenium/standalone-firefox${MOODLE_DOCKER_SELENIUM_SUFFIX}:2.53.1"
     volumes:
       - "${MOODLE_DOCKER_WWWROOT}:/var/www/html:ro"
+  mlbackendpython:
+    image: "moodlehq/moodle-mlbackend-python:2.3.0-python3.7.5"
+    ports:
+      - "5000:5000"
```

Comment:
Hi @dmonllao,
1. you've missed to add a new dependency to the `webserver` service
1. you don't need to expose the TCP port `5000` unless for other purposes: when you use a compose, a dedicated network interconnects all the services within that compose i.e. `mlbackendpython:5000` should be already available without mapping the port on the docker host too

HTH,
Matteo